src/MetricsTimer.h

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This code has been produced by the European Spallation Source
+// and its partner institutes under the BSD 2 Clause License.
+//
+// See LICENSE.md at the top level for license information.
+//
+// Screaming Udder!                              https://esss.se
+
+#pragma once
+#include "Forwarder.h"
```

Comment:
We don't need to include "Forwarder.h" here, you can remove this and forward declare `Forwarder::Converter` instead.

Reply:
@DolicaAkelloEgwel The include for `Converter.h` can go in the implementation file if you forward declare the `Converter` class here. If you are not familiar with forward declaration there is an explanation here: https://arne-mertz.de/2018/03/forward-declarations/